ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/ChainDB/Impl/Types.hs

```diff
@@ -586,7 +587,7 @@ data TraceOpenEvent blk =
   | StartedOpeningVolatileDB
 
     -- | The VolatileDB was opened.
-  | OpenedVolatileDB
+  | OpenedVolatileDB MaxSlotNo
```

Comment:
Should we explain what this `MaxSlotNo` refers to?

Reply:
Sure